wvd-templates/Create and provision WVD host pool/mainTemplate.json

```diff
@@ -250,6 +258,17 @@
                 "name": "[variables('avSetSKU')]"
             }
         },
+        {
+            "type": "Microsoft.Storage/storageAccounts",
+            "sku": {
+                "name": "[variables('storageAccountType')]"
+            },
+            "kind": "Storage",
+            "name": "[toLower(Parameters('storageAccountName'))]",
+            "apiVersion": "2017-06-01",
+            "location": "[resourceGroup().location]",
+            "condition": "[equals(variables('vmTemplateName'), 'unmanagedDisks-galleryvm')]"
+        },
```

Comment:
Most likely need a dependsOn statement from the "vmCreation-linkedTemplate" resource to this